app/web/Dockerfile

```diff
@@ -16,9 +16,12 @@ RUN yarn install --frozen-lockfile
 
 COPY . .
 
-ENV PROTOS_SHA=63b57d8c12b291f8f5c1fd740879530e7da7865797f783dafcda7b3432c2efba
+# ENV PROTOS_SHA=b084f43d1becbe18a256819907a9beeb2008966d47eb9f7786aac49c6c381706
+# ENV PROTOS_URL=https://protos--by-sha.preview.couchershq.org/$PROTOS_SHA/ts.tar.gz
 
-RUN wget https://protos--by-sha.preview.couchershq.org/$PROTOS_SHA/ts.tar.gz
+ENV PROTOS_URL=https://develop--protos.preview.couchershq.org/ts.tar.gz
+
+RUN wget $PROTOS_URL
```

Comment:
why this? (just to understand it)

Reply:
It's not really related to this PR: but it's a bit cumbersome every time people want to change the protos to have to change this hash. I am not sure if I want this though: I like it when they point to a particular version.

Reply:
✅